react-frontend/src/components/Learning/podcast.js

```diff
@@ -0,0 +1,107 @@
+import React, { useContext } from 'react';
+import { Col, Row } from 'react-flexbox-grid';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import dateFormat from 'dateformat';
+
+import BannerSideImage from '../PageElements/Banners/bannerSideImage';
+import PodcastBanner from './podcastBanner';
+import PodcastCollapsedMenu from './podcastCollapsedMenu';
+
+import Query from '../Query';
+import PODCASTS_QUERY from '../../queries/learning/podcasts';
+import { AppConfigContext } from '../../providers/AppConfig';
+
+import { ContentBlockContainer } from '../StyleComponents/pageContent';
+import { SubHeading, SubSubHeading } from '../StyleComponents/headings';
+import { HrefLinkStandalone } from '../StyleComponents/htmlTags';
+
+const podcastImage = require('../../images/pngIllustrations/podcastGray.png')
+  .default;
+
+function Podcast() {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <div>
+      <PodcastBanner />
+      <ContentBlockContainer id="courses">
+        <Row>
+          <Col sm={12}>
+            <SubHeading style={{ display: 'inline-block' }}>
+              Recent Episodes
+            </SubHeading>
+            <HrefLinkStandalone
+              href="https://bcdevexchange.libsyn.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+              style={{ float: 'right' }}
+            >
+              View all {' >'}
+              <FontAwesomeIcon
+                icon={faExternalLinkAlt}
+                style={{ paddingLeft: '5px' }}
+              />
+            </HrefLinkStandalone>
+          </Col>
+        </Row>
+        {/* PODCASTS_QUERY returns the 3 most recent episodes in Strapi  */}
+        <Query query={PODCASTS_QUERY}>
+          {({ data: { podcasts } }) => {
+            console.log(podcasts);
+            return (
+              <div>
+                {podcasts.map((podcast, i) => {
+                  return (
+                    <Row style={{ marginBottom: '16px' }} key={i}>
+                      <Col xs={12} md={4} style={{ height: '200px' }}>
+                        <img
+                          src={strapiMediaUrl + podcast?.CoverImage?.url}
+                          style={{ maxWidth: '100%', height: '100%' }}
```

Comment:
I think `height: 100%` is messing up the images here:

![fine](https://user-images.githubusercontent.com/16946297/124509791-5e754800-dd87-11eb-8a6b-6644766f54e0.png)
![squished](https://user-images.githubusercontent.com/16946297/124509796-603f0b80-dd87-11eb-8535-673ebbb14a36.png)

I'd suggest doing it the other way around: `style={{ height: 'auto', width: '100%' }}`

Reply:
This has some potentially unwanted effects on small screens. I've modified it to use max height and max width to keep the image a useful size